website/docs/r/ce_anomaly_subscription.html.markdown

```diff
@@ -37,6 +37,32 @@ resource "aws_ce_anomaly_subscription" "test" {
 }
 ```
 
+### threshold_expression
```

Comment:
For consistency we should remove the snake case and capitalize.

```suggestion
### Threshold Expression
```

Reply:
Ah thanks for catching! Accepted suggestion